admin/templates/configuration/tmp/configure/sql/QueryMetadata.sql

```diff
@@ -21,8 +21,8 @@ USE `qservMeta` ;
 -- Table `QCzar`
 -- -----------------------------------------------------
 CREATE TABLE IF NOT EXISTS `QCzar` (
-  `cid` INT NOT NULL AUTO_INCREMENT COMMENT 'Czar identifier',
-  `czar` CHAR(63) NOT NULL,
+  `cid` INT UNSIGNED NOT NULL AUTO_INCREMENT COMMENT 'Czar identifier',
```

Comment:
unsigned is specific to mysql, should we (maybe) stick with int to be more cross-dbms compatible?